src/DataDefinitionsBundle/Provider/TraversableImportDataSet.php

```diff
@@ -9,23 +9,27 @@
  * files that are distributed with this source code.
  *
  * @copyright  Copyright (c) 2016-2019 w-vision AG (https://www.w-vision.ch)
- * @license    https://github.com/w-vision/DataDefinitions/blob/master/gpl-3.0.txt GNU General Public License version 3 (GPLv3)
+ * @license    https://github.com/w-vision/DataDefinitions/blob/master/gpl-3.0.txt GNU General Public License version 3
```

Comment:
Unrelated change?

Reply:
My editor changes this. I'll change it back